packages/@aws-cdk/aws-iotevents/lib/index.ts

```diff
@@ -1,2 +1,9 @@
+export * from "./Input";
+export * from "./DetectorModel";
+
+export * from "./Event";
+export * from "./TransitionEvent";
+export * from "./State";
+
 // AWS::IoTEvents CloudFormation Resources:
-export * from './iotevents.generated';
+export * from "./iotevents.generated";
```

Comment:
IIRC single quotes is consistent with the rest of the repo

Reply:
Is this required? It seems I'm incompetent enough though to be able to configure vscode and prettier to NOT convert single quotes to double quotes.
Also I see that aws-dynamodb-global is using double quotes :-P